WordPress/Sniffs/VIP/SuperGlobalInputUsageSniff.php

```diff
@@ -21,9 +21,32 @@
  * @since   0.3.0
  * @since   0.4.0  This class now extends WordPress_Sniff.
  * @since   0.13.0 Class name changed: this class is now namespaced.
+ *
+ * @deprecated 1.0.0  This sniff has been deprecated.
+ *                    This file remains for now to prevent BC breaks.
  */
 class SuperGlobalInputUsageSniff extends Sniff {
 
+	/**
+	 * If true, an error will be thrown; otherwise a warning.
+	 *
+	 * @var bool
+	 */
+	public $error = true;
+
+	/**
+	 * Keep track of whether the warnings have been thrown to prevent
+	 * the messages being thrown for every token triggering the sniff.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var array
+	 */
+	private $thrown = array(
+		'DeprecatedSniff'                 => false,
+		'FoundPropertyForDeprecatedSniff' => false,
```

Comment:
This sniff does not have any public properties which can be changed, so the `FoundPropertyForDeprecatedSniff` entry can be removed.